test.js

```diff
@@ -293,3 +293,10 @@ test('is.typedArray', t => {
 	t.false(m.typedArray([]));
 	t.false(m.typedArray({}));
 });
+
+test('is.inRange', t => {
+	const x = 3;
+	const arr = [0, 5];
```

Comment:
You can just put this inline in the assertion.

I'd also like to see assertions for `[5, 0]`, `[-5, 5]`, `[5, -5]`.

Reply:
Also add assertions for `is.inRange` that return `false`.

I also agree with inlining the test data.